src/System.Globalization/tests/CultureInfo/CultureInfoCurrentCulture.cs

```diff
@@ -57,6 +57,7 @@ public void CurrentUICulture()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Thread cultures is not honored in UWP.")]
```

Comment:
maybe but that will make it difficult to anyone writing portable library as will always need to catch such exception.